src/WWTMVC5/WwtWebHttpHandler.cs

```diff
@@ -0,0 +1,186 @@
+﻿using Microsoft.Extensions.DependencyInjection;
```

Comment:
@twsouthwick what type of testing did you have in mind for this?  I really like where this change is taking us - but I'm curious if you're validating this within our solution or by using the URL DB Python code

Reply:
Just within the solution. The URL DB Python code would be a good test.